### PR TITLE
[security] prevent XSS on FAB list views

### PR DESCRIPTION
**Pull request by mistercrunch**
@jefffeng @ascott @bkyryliuk

### CODE REVIEW

File: caravel/models.py [hunk 1]
```diff
@@ -182,7 +183,7 @@ def datasource_link(self):
         if self.table:
             return self.table.link
         elif self.druid_datasource:
-            return self.druid_datasource.link
```
Comment (bkyryliuk) [inline caravel/models.py]: is it intended?

**Comment by bkyryliuk** [comment]
Looks good, just a minor comment, looks like `return self.druid_datasource.link` should stay.

File: caravel/models.py [hunk 2]
```diff
@@ -102,12 +102,13 @@ def changed_by_(self):
...
     def changed_on_(self):
-        return '<span class="no-wrap">{}</span>'.format(self.changed_on)
+        return Markup(
+            '<span class="no-wrap">{}</span>'.format(self.changed_on))
 
     @renders('changed_on')
     def modified(self):
         s = humanize.naturaltime(datetime.now() - self.changed_on)
-        return '<span class="no-wrap">{}</nobr>'.format(s)
+        return Markup('<span class="no-wrap">{}</nobr>'.format(s))
```
Comment (xrmx) [inline caravel/models.py]: what is this </nobr>?

Comment (bkyryliuk) [inline caravel/models.py]: http://jinja.pocoo.org/docs/dev/api/#jinja2.Markup

**Comment by bkyryliuk** [comment]
LGTM

**Pull request merged**